StreamVideoTests/Utilities/Dummy/ScreensharingSettings+Dummy.swift

```diff
@@ -0,0 +1,15 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import StreamVideo
+
+extension ScreensharingSettings {
```

Comment:
For some of these (like this one) feels like an overkill to have a separate file and a `dummy` method.

Reply:
It's easier to scan the project and see what has a mock and IMO they are clearer. I don't really understand the term overkill as those files are in tests under a specific folder named Mocks and they don't complicate something.

Reply:
For me, it's an overkill to create new files and extensions just for passing one default bool parameter. For larger initializers it makes sense, but not for most of these. 
In any case, up to you.

Reply:
I think that maintaining seperate files per file and also keeping the `dummy` format we have on Chat is benefitial. While looking into the `MockResponseBuilder` i realised that the file is also shared between Video and VideoUI only by Xcode settings, which means that the Package itself will fail to run tests. 
As a final improvement i believe we should move those mocks in to a seperate TestHelpers target as we do for Chat.

Reply:
Yes, good point, we should create a new target for these.